config/config/common-config.yaml

```diff
@@ -916,6 +916,8 @@ trivy:
   # configurations for an offline / air-gapped environment
   # ref: https://github.com/aquasecurity/trivy-operator/tree/main/deploy/helm#values
   scanner:
+    resources: {}
+    timeout: ""
```

Comment:
How does it behave when you set the timeout to empty string? I see the default is 5 minutes in the values file.

Reply:
with `scanner.timeout: ""`:

```
➜ ck8s-apps ops helmfile wc -l name=trivy-operator template | grep timeout
  trivy.timeout: "5m0s"
```

with `scanner.timeout: "1337m"`:
  
```
➜ ck8s-devbox ck8s-apps ops helmfile wc -l name=trivy-operator template | grep -i "timeout"
  trivy.timeout: "1337m"
```

Reply:
Sweet!